[CS2103T-T12-3] Athletick

hellodommy opened the pull request:
Step 7 of [repo setup](https://nus-cs2103-ay1920s1.github.io/website/schedule/week6/project.html).
@ChangUo79 @jeunhoe @junhuplim @shawnlsj97

danielwys commented:
**User stories**
1. Is the user story format correct?: Yes
2. All three parts present?: Yes
3. Does the benefit match the function?: Yes
4. Any important user stories missing?: Yes, insufficient user stories (perhaps 15?) - program seems like it involves only light modifications to AB3.

**Use Cases**
General Issues:
1. Marking attendance of players: No indication of what marking attendance is like (session based? Date based?)
2. Key in Performance of a Player: What is a ‘performance’? Is it an object created in advanced and hence ‘tagged’ onto a user? Maybe it should be “System requests coach to key in performance” or “System requests coach to select performance grade.”
3. No corresponding use case to "Coach who wants fit players | Filter players by overweight BMIs | Single them out and get them to lose weight" - no mention of input of height/weight for BMI calculation.

1. Is the step numbering correct?: Yes
2. Are all possible extensions mentioned?: Yes
3. Are there any unnecessary UI details mentioned?: No.
4. Any other formatting/notational errors?: Not following the Use Case formatting - no System, Use Case number, Actor and Precondition.

**Non-functional Requirements**
1. Are the NFRs are really Non-Functional?: Yes
2. Is each NFR well-defined (i.e., possible to decide when it has been met)?: Yes
3. Is each NFR reasonably achievable?: Yes
4. Are more relevant NFRs have been left out?: Yes - No additional NFRs beyond given NFRs from stock AB3 DG.

hellodommy committed:
Update Storage class diagram with Performance and Training components

hellodommy committed:
Describe Storage responsibilities

hellodommy committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main into developer-guide

hellodommy committed:
Merge pull request #232 from hellodommy/developer-guide

DG: Update Storage and Performance sections

jeunhoe committed:
Update PPP, UG and DG

junhuplim committed:
Update PPP

junhuplim committed:
Update PPP

junhuplim committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main

* 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main:
  Describe Storage responsibilities
  Update Storage class diagram with Performance and Training components
  Add design consideration for using HashMap
  Add first design consideration

jeunhoe committed:
Comply with checkstyle

junhuplim committed:
Merge pull request #234 from junhuplim/master

Update PPP

jeunhoe committed:
Merge pull request #233 from jeunhoe/documents-branch

Update PPP, UG and DG

junhuplim committed:
update junhuplim.adoc

junhuplim committed:
Update junhuplim.adoc

junhuplim committed:
Update junhuplim.adoc

junhuplim committed:
Update junhuplim.adoc

junhuplim committed:
update dg diagram

junhuplim committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main

* 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main:
  Update junhuplim.adoc
  Update junhuplim.adoc
  Update junhuplim.adoc
  update junhuplim.adoc
  Comply with checkstyle
  Update PPP, UG and DG

# Conflicts:
#	docs/team/junhuplim.adoc

junhuplim committed:
resolve merge conflict

junhuplim committed:
Merge pull request #235 from junhuplim/master

Update diagram size

junhuplim committed:
dg size

junhuplim committed:
Merge pull request #236 from junhuplim/master

dg size

jeunhoe committed:
Move PPP to correct directory

jeunhoe committed:
Merge branch 'master' of https://github.com/jeunhoe/main into documents-branch

* 'master' of https://github.com/jeunhoe/main:
  dg size
  resolve merge conflict
  update dg diagram
  Update junhuplim.adoc
  Update junhuplim.adoc
  Update junhuplim.adoc
  update junhuplim.adoc
  Update PPP
  Update PPP
  Describe Storage responsibilities
  Update Storage class diagram with Performance and Training components
  Add design consideration for using HashMap
  Add first design consideration
  Update diagram labelling
  Update PPP
  Update ug
  Fix accidental spacing
  Add sequence diagram for viewing records
  Introduce Performance in DG

jeunhoe committed:
Edit UserGuide

jeunhoe committed:
Update DG

jeunhoe committed:
Merge pull request #237 from jeunhoe/documents-branch

Move PPP to correct directory

hellodommy committed:
Add reference frame for viewing records

hellodommy committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main into developer-guide

hellodommy committed:
Elaborate on aspect of design consideration

hellodommy committed:
Add implementation for FilterCommand

shawnlsj97 committed:
Update DG

hellodommy committed:
Fix trailing whitespace on Junhup's PPP

hellodommy committed:
Merge pull request #238 from hellodommy/developer-guide

Developer guide

hellodommy committed:
Elaborate on deleting event

hellodommy committed:
Update screenshots for adding and deleting Event

hellodommy committed:
Add guide for deleting records

hellodommy committed:
Update guide for viewing Events

hellodommy committed:
Update guide for viewing Records

hellodommy committed:
Refer to diagrams in guide

hellodommy committed:
Merge pull request #239 from hellodommy/user-guide

User guide

junhuplim committed:
Update UG/DG

junhuplim committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main

* 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main:
  Refer to diagrams in guide
  Update guide for viewing Records
  Update guide for viewing Events
  Add guide for deleting records
  Update screenshots for adding and deleting Event
  Elaborate on deleting event
  Fix trailing whitespace on Junhup's PPP
  Add implementation for FilterCommand
  Elaborate on aspect of design consideration
  Add reference frame for viewing records
  Update DG
  Edit UserGuide
  Move PPP to correct directory

# Conflicts:
#	docs/team/junhuplim.adoc

junhuplim committed:
merge conflict

hellodommy committed:
Update Filter screenshots

junhuplim committed:
update Icon

hellodommy committed:
Update screenshots for Find

junhuplim committed:
checkstyle

junhuplim committed:
comply checkstyle

hellodommy committed:
Merge pull request #241 from hellodommy/user-guide

User guide

junhuplim committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main

* 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main:
  Update screenshots for Find
  Update Filter screenshots

junhuplim committed:
Remove htmls

junhuplim committed:
Merge pull request #242 from junhuplim/master

Update UG/DG Icon

junhuplim committed:
Try out Box

shawnlsj97 committed:
Test DG formatting

shawnlsj97 committed:
Revert DG changes

shawnlsj97 committed:
Update DeveloperGuide.adoc

jeunhoe committed:
Edit UG, DG and PPP

junhuplim committed:
Update Icon and alignment

junhuplim committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main

* 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main:
  Update DeveloperGuide.adoc
  Revert DG changes
  Test DG formatting

junhuplim committed:
Update DG alignment

junhuplim committed:
Merge pull request #243 from junhuplim/master

Update UG DG Icon and alignment

jeunhoe committed:
Merge pull request #244 from jeunhoe/documents-branch

Edit UG, DG and PPP

junhuplim committed:
Update PPP

junhuplim committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main

* 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main:
  Edit UG, DG and PPP

junhuplim committed:
Merge pull request #245 from junhuplim/master

Update PPP

jeunhoe committed:
Tag symbols for use in PPP

jeunhoe committed:
Revert "Tag symbols for use in PPP"

This reverts commit 70d0940ad6fa43e88b41a3b2dcdd79d4a0fdc7fa.

jeunhoe committed:
Edit PPP and tagged symbol in DG

jeunhoe committed:
Merge branch 'master' into documents-branch

jeunhoe committed:
Add tags to UG for PPP

jeunhoe committed:
Merge branch 'documents-branch' of https://github.com/jeunhoe/main into documents-branch

* 'documents-branch' of https://github.com/jeunhoe/main:
  Update PPP

jeunhoe committed:
Tag DG for PPP

jeunhoe committed:
Edit PPP

shawnlsj97 committed:
Update DG

shawnlsj97 committed:
Update DeveloperGuide.adoc

junhuplim committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main

* 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main:
  Update DeveloperGuide.adoc

junhuplim committed:
UG Test out

junhuplim committed:
Merge pull request #247 from junhuplim/master

UP Test

junhuplim committed:
Remove changes

junhuplim committed:
Merge pull request #248 from junhuplim/master

Remove changes Test

hellodommy committed:
Add introduction for PPP

shawnlsj97 committed:
Update DeveloperGuide.adoc

shawnlsj97 committed:
Update UserGuide.adoc

junhuplim committed:
Test out UG/DG format

junhuplim committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main

* 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main:
  Update UserGuide.adoc
  Update DeveloperGuide.adoc

# Conflicts:
#	docs/UserGuide.adoc

junhuplim committed:
Resolve merge conflict

junhuplim committed:
Merge pull request #249 from junhuplim/master

Test out UG/DG format

junhuplim committed:
Update PPP

junhuplim committed:
Update PPP

hellodommy committed:
Add summary of contributions for PPP

junhuplim committed:
Update PPP

junhuplim committed:
Update PPP

jeunhoe committed:
Merge pull request #246 from jeunhoe/documents-branch

Documents branch

hellodommy committed:
Add extracts from User Guide and Developer Guide

shawnlsj97 committed:
Update PPP

junhuplim committed:
checkstyle

shawnlsj97 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main

# Conflicts:
#	docs/DeveloperGuide.adoc

hellodommy committed:
Resolve merge conflict with Jeunhoe's DG tags

Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main into portfolio

# Conflicts:
#	docs/DeveloperGuide.adoc

junhuplim committed:
Merge pull request #250 from junhuplim/master

Update PPP

hellodommy committed:
Merge pull request #251 from hellodommy/portfolio

PPP for Dominique

shawnlsj97 committed:
Update checkstyle

hellodommy committed:
Link to PPP in About Us

hellodommy committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main into portfolio

shawnlsj97 committed:
Update PPP

shawnlsj97 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main

hellodommy committed:
Reorder team management in PPP

hellodommy committed:
Merge pull request #252 from hellodommy/portfolio

Portfolio

shawnlsj97 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main

shawnlsj97 committed:
Update DG and UG tags

shawnlsj97 committed:
Merge pull request #231 from shawnlsj97/master

Update CommandResult

hellodommy committed:
Fix direction of arrow in activity diagram

hellodommy committed:
Include extracts from User Guide

hellodommy committed:
Include design considerations from DG in PPP

hellodommy committed:
Include introduction to Performance in DG into PPP

hellodommy committed:
Include overview of Performance from DG into PPP

hellodommy committed:
Resize class diagrams in DG

junhuplim committed:
Update PPP

hellodommy committed:
Resize activity diagram from DG

junhuplim committed:
Update PPP

junhuplim committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main

* 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main:
  Update DG and UG tags
  Reorder team management in PPP
  Update PPP
  Link to PPP in About Us
  Update checkstyle
  Update PPP
  Add extracts from User Guide and Developer Guide
  Add summary of contributions for PPP
  Add introduction for PPP
  Update DG
  Edit PPP
  Tag DG for PPP
  Add tags to UG for PPP
  Edit PPP and tagged symbol in DG
  Revert "Tag symbols for use in PPP"
  Tag symbols for use in PPP
  Update DG
  Update DG
  Update checkstyle
  Update CommandResult

junhuplim committed:
Merge pull request #253 from junhuplim/master

Update PPP

ChangUo79 committed:
Updated ppp, ug and dg

junhuplim committed:
Update final draft PPP

junhuplim committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main

* 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main:

junhuplim committed:
Merge pull request #255 from junhuplim/master

Update final PPP

ChangUo79 committed:
Merge branch 'master' of https://github.com/ChangUo79/main into Developer-user-guide

# Conflicts:
#	docs/DeveloperGuide.adoc

jeunhoe committed:
Edit diagrams for Logic, Model and Training in DG

jeunhoe committed:
Merge pull request #257 from jeunhoe/documents-branch

Edit diagrams for Logic, Model and Training in DG

ChangUo79 committed:
Merge pull request #256 from ChangUo79/Developer-user-guide

Developer user guide

jeunhoe committed:
Edit PPP

jeunhoe committed:
Edit PPP links

ChangUo79 committed:
Updated ppp and AboutUs.adoc

ChangUo79 committed:
Merge pull request #258 from ChangUo79/Developer-user-guide

Updated ppp and AboutUs.adoc

ChangUo79 committed:
Updated to remove hyperlink when generating pdf

ChangUo79 committed:
Merge pull request #259 from ChangUo79/Developer-user-guide

Updated to remove hyperlink when generating pdf

jeunhoe committed:
Edit PPP

jeunhoe committed:
Merge branch 'master' into documents-branch

jeunhoe committed:
Remove unused code and update test methods

hellodommy committed:
Update class and activity diagrams in DG

hellodommy committed:
Merge pull request #254 from hellodommy/portfolio

Portfolio

hellodommy committed:
Resize activity diagram in DG

jeunhoe committed:
Merge pull request #261 from jeunhoe/test-branch

Remove unused code and update test methods

jeunhoe committed:
Merge pull request #260 from jeunhoe/documents-branch

Edit PPP

hellodommy committed:
Finalise extracts from DG into PPP

shawnlsj97 committed:
Update DG and PPP

shawnlsj97 committed:
Update checkstyle

shawnlsj97 committed:
Merge pull request #262 from shawnlsj97/master

Update DG and PPP

hellodommy committed:
Add link to GitHub code contributed in PPP

ChangUo79 committed:
updated ppp

ChangUo79 committed:
Merge branch 'master' of https://github.com/ChangUo79/main into Developer-user-guide

hellodommy committed:
Resize UG Performance screenshots

ChangUo79 committed:
Updated to fix display issue

hellodommy committed:
Update screenshots and documentation for delete -p

ChangUo79 committed:
Merge pull request #263 from ChangUo79/Developer-user-guide

Developer user guide

hellodommy committed:
Merge pull request #264 from hellodommy/portfolio

Portfolio

shawnlsj97 committed:
Update DG and PPP

shawnlsj97 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main

hellodommy committed:
Update Ui and Gui screenshots

hellodommy committed:
Remove Gui with incorrect naming

hellodommy committed:
Add Gui with correct extension naming

junhuplim committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main

* 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main: (25 commits)
  Update screenshots and documentation for delete -p
  Updated to fix display issue
  Resize UG Performance screenshots
  updated ppp
  Add link to GitHub code contributed in PPP
  Update checkstyle
  Update DG and PPP
  Finalise extracts from DG into PPP
  Resize activity diagram in DG
  Update class and activity diagrams in DG
  Remove unused code and update test methods
  Edit PPP
  Updated to remove hyperlink when generating pdf
  Updated ppp and AboutUs.adoc
  Edit PPP links
  Edit PPP
  Edit diagrams for Logic, Model and Training in DG
  Updated ppp, ug and dg
  Resize activity diagram from DG
  Resize class diagrams in DG
  ...

hellodommy committed:
Merge pull request #265 from hellodommy/portfolio

Update Ui and Gui screenshots

junhuplim committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main

* 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main:
  Add Gui with correct extension naming
  Remove Gui with incorrect naming
  Update Ui and Gui screenshots

junhuplim committed:
Update UG Diagrams

jeunhoe committed:
Edit PPP

ChangUo79 committed:
Added diagram

jeunhoe committed:
Revert "Edit PPP"

This reverts commit 43084aef1761c05617c6e77cbb846b573685558e.

junhuplim committed:
Merge pull request #266 from junhuplim/master

Update UG Diagram

ChangUo79 committed:
minor changes

jeunhoe committed:
Edit PPP

ChangUo79 committed:
Merge pull request #267 from ChangUo79/Developer-user-guide

Added sequence diagram

hellodommy committed:
Include links to contributions in UG and DG into PPP

jeunhoe committed:
Merge pull request #268 from jeunhoe/documents-branch

Finalise changes to PPP

junhuplim committed:
Update PPP

junhuplim committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main

* 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main:
  Edit PPP
  minor changes
  Revert "Edit PPP"
  Added diagram
  Edit PPP

hellodommy committed:
Fix grammar mistakes

junhuplim committed:
Merge pull request #270 from junhuplim/master

Update PPP

hellodommy committed:
Improve clarity on enhancement added in PPP

hellodommy committed:
Restore enhancement to filter in PPP

hellodommy committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main into portfolio

junhuplim committed:
Update DG size

junhuplim committed:
Merge pull request #271 from junhuplim/master

Update DG size

hellodommy committed:
Resize images in DG

hellodommy committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main into portfolio

junhuplim committed:
Update UG size

junhuplim committed:
Merge pull request #272 from junhuplim/master

Update UG size

hellodommy committed:
Shorten lines to fit 10 pages in PPP

hellodommy committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main into portfolio

junhuplim committed:
Update PPP

hellodommy committed:
Standardise syntax in PPP

junhuplim committed:
Merge pull request #273 from junhuplim/master

Update PPP

hellodommy committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main into portfolio

hellodommy committed:
Add full name in About Us

hellodommy committed:
Merge pull request #269 from hellodommy/portfolio

Include links to contributions in UG and DG into PPP

junhuplim committed:
Last PPP

junhuplim committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main

* 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main:
  Standardise syntax in PPP
  Shorten lines to fit 10 pages in PPP
  Resize images in DG
  Restore enhancement to filter in PPP
  Improve clarity on enhancement added in PPP
  Fix grammar mistakes
  Include links to contributions in UG and DG into PPP

shawnlsj97 committed:
Fix resizing for infodisplay

junhuplim committed:
Merge pull request #274 from junhuplim/master

Last PPP

hellodommy committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main into portfolio

shawnlsj97 committed:
Merge pull request #275 from shawnlsj97/master

Fix resizing for infodisplay

hellodommy committed:
Fix broken links to key contributions in UG and DG

jeunhoe committed:
Edit PPP and DG

jeunhoe committed:
Merge pull request #277 from jeunhoe/developer-guide-branch

Edit PPP and DG

ChangUo79 committed:
Updated with some minor changes

jeunhoe committed:
Edit DG

hellodommy committed:
Standardise use of mark-up

jeunhoe committed:
Merge pull request #279 from jeunhoe/developer-guide-branch

Fixed grammar errors in PPP

hellodommy committed:
Fix grammatical error in PPP

hellodommy committed:
Fix minor grammatical error in PPP

ChangUo79 committed:
Merge pull request #278 from ChangUo79/Developer-user-guide

Updated with some minor changes

junhuplim committed:
Review Changes

junhuplim committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main

* 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main:
  Edit DG
  Updated with some minor changes
  Edit PPP and DG
  Fix resizing for infodisplay
  Update DG and PPP

junhuplim committed:
Merge pull request #280 from junhuplim/master

Review Changes

hellodommy committed:
Fix inconsistent letter casing

hellodommy committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main into portfolio

hellodommy committed:
Merge pull request #276 from hellodommy/portfolio

Portfolio

hellodommy committed:
Add Instructions for Manual Testing for deleting an event

shawnlsj97 committed:
Update UG, DG and PPP

hellodommy committed:
Clean up Instructions for Manual Testing

junhuplim committed:
Manual Testing

junhuplim committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main

* 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main:
  Fix inconsistent letter casing
  Fix minor grammatical error in PPP
  Fix grammatical error in PPP
  Standardise use of mark-up
  Fix broken links to key contributions in UG and DG
  Add full name in About Us

hellodommy committed:
Add product survey

hellodommy committed:
Fix con in product survey

shawnlsj97 committed:
Merge pull request #282 from shawnlsj97/master

Update UG, DG and PPP

junhuplim committed:
Merge pull request #283 from junhuplim/master

Manual Testing

hellodommy committed:
Add Instructions for Manual Testing for adding an event

hellodommy committed:
Fix merge conflict with Junhup's DG changes
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main into portfolio

# Conflicts:
#	docs/DeveloperGuide.adoc

hellodommy committed:
Fix newline error in DG

junhuplim committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main

* 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main:
  Update UG, DG and PPP

hellodommy committed:
Merge pull request #281 from hellodommy/portfolio

Add Instructions for Manual Testing for deleting an event

junhuplim committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main

* 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main:
  Fix newline error in DG
  Add Instructions for Manual Testing for adding an event
  Fix con in product survey
  Add product survey
  Clean up Instructions for Manual Testing
  Add Instructions for Manual Testing for deleting an event

junhuplim committed:
Update Aboutus

junhuplim committed:
Merge pull request #284 from junhuplim/master

Update aboutus

ChangUo79 committed:
Updated Manual testing

ChangUo79 committed:
Merge branch 'master' of https://github.com/ChangUo79/main into Developer-user-guide

# Conflicts:
#	docs/DeveloperGuide.adoc

ChangUo79 committed:
Merge pull request #285 from ChangUo79/Developer-user-guide

Developer user guide

shawnlsj97 committed:
Update manual testing

shawnlsj97 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main

# Conflicts:
#	docs/DeveloperGuide.adoc

shawnlsj97 committed:
Update display pic

shawnlsj97 committed:
Merge pull request #286 from shawnlsj97/master

Update manual testing

hellodommy committed:
Make manual testing headers discrete

hellodommy committed:
Remove address book from About Us

hellodommy committed:
Merge pull request #287 from hellodommy/master

Make manual testing headers discrete

shawnlsj97 committed:
Fix 1 error line in code. Email prof Damith regarding this change.

shawnlsj97 committed:
Merge pull request #288 from shawnlsj97/master

Fix 1 error line in code. Emailed prof Damith regarding this change.

ChangUo79 committed:
changing of filename to lowercase

ChangUo79 committed:
Merge pull request #290 from ChangUo79/Developer-user-guide

changing of filename to lowercase

shawnlsj97 committed:
Update personal website

shawnlsj97 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-3/main

shawnlsj97 committed:
Merge pull request #291 from shawnlsj97/master

Update aboutus
